src/entities.lua

```diff
@@ -6,19 +6,30 @@ local m_animation = require"src/animation"
 --- Draws an entity.
 --- @param self Entity The entity to draw.
 local function draw(self)
+	local sprite,pos,flipped = self:unpack_draw_data()
+	spr(
+		sprite,
+		pos.x,pos.y,
+		flipped
+	)
```

Comment:
This could be one-lined.
```suggestion
	spr(sprite,pos.x,pos.y,flipped)
```